options.rb

```diff
@@ -103,6 +139,32 @@ def add_album
     puts '----------------------------------------------'
   end
 
+  def add_book
+    print 'Publisher: '
+    publisher = gets.chomp
+
+    while true
+      print 'Cover state (select 1 for "good" or 2 for "bad" ): '
+      cover_option = gets.chomp.to_i
+
+      case cover_option
+      when 1 then cover_state = 'good'
+      when 2 then cover_state = 'bad'
+      else
+        puts "Wrong option \n\n"
+        next
+      end
+      break
+    end
+
+    print 'Publish date (YYYY-MM-DD): '
+    publish_date = gets.chomp
+
+    book = Book.new(publisher: publisher, cover_state: cover_state, publish_date: publish_date)
```

Comment:
Before finishing the book creation, I think it would be a good idea to pass it through the Decorator module. This way every book item will have a label, a genre and an author. You can do that by using this code: `Decorator.decorate(album, @authors, @genres, @labels)`